src/js/_store/modules/categories.js

```diff
@@ -29,5 +44,20 @@ export default {
         commit('failRequest', { message: err.message });
       });
     },
+    createCategory({ commit, rootGetters }, categoryValue) {
```

Comment:
[nits]

ぱっと見何の値が引数に入ってくるのかがわかりづらいので、以下のようにして良いかなと思います。
createCategory() の引数で受け取る `name` なので、「カテゴリー名」ということは伝わるかなと思います。

```suggestion
    createCategory({ commit, rootGetters }, name) {
```